editor.planx.uk/src/components/Feedback/FeedbackForm.test.tsx

```diff
@@ -0,0 +1,147 @@
+import { waitFor } from "@testing-library/react";
+// eslint-disable-next-line no-restricted-imports
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import { axe, setup } from "testUtils";
+
+import { FeedbackFormInput } from ".";
+import FeedbackForm from "./FeedbackForm";
+
+const mockHandleSubmit = jest.fn();
+
+const mockUnlabeledInput: FeedbackFormInput[] = [
+  {
+    name: "userComment",
+    ariaDescribedBy: "comment-title",
+  },
+];
+
+const mockLabeledInputs: FeedbackFormInput[] = [
+  { name: "userContext", id: "userContext", label: "User Context" },
+  { name: "userComment", id: "userComment", label: "User Comment" },
+];
+
+describe("FeedbackForm functionality", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test("renders unlabelled input correctly", () => {
+    const { getByLabelText } = setup(
+      <FeedbackForm
+        inputs={mockUnlabeledInput}
+        handleSubmit={mockHandleSubmit}
+      />,
+    );
+    const renderedInput = getByLabelText("Leave your feedback");
+    expect(renderedInput).toBeInTheDocument();
+  });
+
+  test("renders labelled inputs correctly", () => {
+    const { getByLabelText } = setup(
+      <FeedbackForm
+        inputs={mockLabeledInputs}
+        handleSubmit={mockHandleSubmit}
+      />,
+    );
+    const contextInput = getByLabelText("User Context");
+    const commentInput = getByLabelText("User Comment");
+    expect(contextInput).toBeInTheDocument();
+    expect(commentInput).toBeInTheDocument();
+  });
+
+  test("renders the feedback disclaimer with the input", () => {
+    const { getByText } = setup(
+      <FeedbackForm
+        inputs={mockUnlabeledInput}
+        handleSubmit={mockHandleSubmit}
+      />,
+    );
+    expect(
+      getByText(
+        /Do not share personal or financial information in your feedback./i,
+      ),
+    ).toBeInTheDocument();
+  });
+
+  test("can submit an unlabelled input", async () => {
+    const { getByLabelText, getByText } = setup(
+      <FeedbackForm
+        inputs={mockUnlabeledInput}
+        handleSubmit={mockHandleSubmit}
+      />,
+    );
+    await userEvent.type(
+      getByLabelText("Leave your feedback"),
+      "This is a test comment",
+    );
+    await userEvent.click(getByText("Send feedback"));
+
+    await waitFor(() => {
```

Comment:
I think the `await waitFor(() => {})` wrapper is redundant - do tests pass without it?